components/CommentList.jsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import CommentCard from "./CommentCard";
+
+const CommentList = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  const [loadingComments, setLoadingComments] = useState(true);
+
+  useEffect(() => {
```

Comment:
I would consider moving all API calls into a separate file and exporting them for use in you components - this makes you code a lot cleaner.

Reply:
Smashed it 👍